### ISSUE TITLE
Support evaluation of TRTLLM checkpoints on logprob benchmarks

### ISSUE THREAD

**Issue by athitten** [NVIDIA-NeMo/Evaluator#368]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by athitten** [comment]
From prior experience with TRTLLM, although generate can return logprobs they could not be used correctly for evaluation. Need to revisit this and verify. If not feasible might have to add code to retrieve the logprobs from logits. TRTLLM's generate can return both    generation and context logits.

**Comment by athitten** [comment]
Closing this, since its not a priority at the moment. Will open a new issue for future
